samples/storage_csm_powerscale_v280.yaml

```diff
@@ -395,3 +433,42 @@ spec:
             # Default value: "otel-collector:55680"
             - name: "COLLECTOR_ADDRESS"
               value: "otel-collector:55680"
+    - name: resiliency
+      # enabled: Enable/Disable Resiliency feature
+      # Allowed values:
+      #   true: enable Resiliency feature(deploy podmon sidecar)
+      #   false: disable Resiliency feature(do not deploy podmon sidecar)
+      # Default value: false
+      enabled: false
+      configVersion: v1.6.0
```

Comment:
Hi @chimanjain Can you please confirm the configVersion of resiliency module?
I think it should be v1.7.0 i.e. latest one that we will be releasing in this quarter.

Reply:
Done